rbac/server/api/packs.py

```diff
@@ -40,6 +41,39 @@
 
 
 @PACKS_BP.get("api/packs")
+@doc.summary("API Endpoint to get all packs")
+@doc.description("API Endpoint to get all packs.")
+@doc.consumes({"head": str}, location="query")
+@doc.consumes({"limit": int}, location="query")
+@doc.consumes({"start": int}, location="query")
+@doc.produces(
+    {
+        "data": {
```

Comment:
shouldn't `data` hold a list?

Reply:
Good catch, I've updated it to a list now.